src/Database/Attach/File.php

```diff
@@ -86,8 +86,11 @@ public function fromPost($uploadedFile)
         $this->file_size = $uploadedFile->getClientSize();
         $this->content_type = $uploadedFile->getMimeType();
         $this->disk_name = $this->getDiskName();
-
-        $this->putFile($uploadedFile->getRealPath(), $this->disk_name);
+        $realpath = ( empty(trim($uploadedFile->getRealPath())) 
```

Comment:
I assume this is because the file actually does not exist, I assume `getRealPath` is the same as `realpath($uploadedFile->getPath() . DIRECTORY_SEPARATOR . $uploadedFile->getFileName())` which would rightfully return null if the file didn't exist. Can you confirm if the `realpath` code works too?